packages/platform/src/services/GameService.ts

```diff
@@ -925,7 +925,22 @@ export async function updatePlayerData(
 }
 
 export async function getGames(args, ctx: Context) {
-  return ctx.prisma.game.findMany()
+  let result = await ctx.prisma.game.findMany({
```

Comment:
```suggestion
  const result = await ctx.prisma.game.findMany({
```

always use const unless it needs to be changed somewhere (is normally also highlighted by the linter)

Reply:
Good catch thanks! I thought I did a modification and forgot to change it back.